beta/tests/sota_checkpoints_eval.json

```diff
@@ -149,7 +149,8 @@
                     "target": 33.44,
                     "weights": "retinanet/retinanet.h5",
                     "metric_type": "mAP",
-                    "model_description": "retinanet"
+                    "model_description": "retinanet",
+                    "batch": 40
```

Comment:
What reason do you specify batch size = 40?

Reply:
@alexsu52 I tested it locally on configuration from CI and this was the max batch